example/src/App.tsx

```diff
@@ -186,6 +192,11 @@ export default function App() {
           component={BasicPlayback}
           options={{ title: 'Basic playback' }}
         />
+        <RootStack.Screen
+          name="BasicTvPlayback"
+          component={BasicTvPlayback}
+          options={{ title: 'Basic TV playback' }}
+        />
```

Comment:
What about having this on the list only when running on an Android TV?
Especially as this has no useful effect on other platforms.

```suggestion
        {Platform.OS === 'android' && Platform.isTV && (
          <RootStack.Screen
            name="BasicTvPlayback"
            component={BasicTvPlayback}
            options={{ title: 'Basic TV playback' }}
          />
        )}
```

Reply:
Good idea!

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/550/commits/4a6e7ab222ca937df91c0d67be06c4de9e81e9b8